pagopa/pn-frontend
# wrong fiscal code message - pn-4237

## Short description
Show specific message when user tries to create a delegation with wrong tax id code 

## List of changes proposed in this pull request
- Added new class to manage the PN_GENERIC_INVALIDPARAMETER_PATTERN error code

## How to test
Create a delegate with tax id code TMOCHR95S51H501H and check that "Il codice fiscale inserito non è valido" message is shown
